opentelemetry-sdk/tests/metrics/test_aggregation.py

```diff
@@ -88,86 +79,7 @@ def test_collect(self):
         )
 
 
-class TestAsynchronousSumAggregation(TestCase):
```

Comment:
We need to have some test coverage still for the `SumAggregation(True, AggregationTemporality.CUMULATIVE)` case.

Reply:
Added test cases